tests/win_bindings.rs

```diff
@@ -0,0 +1,20 @@
+#![cfg(all(windows, feature = "clock", feature = "std"))]
+
+use std::fs::OpenOptions;
```

Comment:
I prefer not to import `io::Result`, instead importing `io::self` and using `io::Result<()>`.